### ISSUE TITLE
[Feature] A new Plugin for Odysee Like the Youtube one

### ISSUE THREAD

**Issue by andreas-fahrecker** [rNeomy/auto-tab-discard#390]
A new Plugin for [Odysee](https://odysee.com/), that saves the timestamp just like the YouTube Plugin, would be very nice.

**Comment by andreas-fahrecker** [comment]
I could write a Pull-Request for that myself, but I am unsure how should structure it.

Should I rather create a new Plugin like I suggested in my initial comment on the Issue. Or should I add the functionality to the existing YouTube plugin?

**Comment by rNeomy** [comment]
Yes, lets have things separated. Create a new plugin and request pull merge.
